web/src/search/input/Suggestion.tsx

```diff
@@ -96,7 +96,7 @@ export function createSuggestion(item: GQL.SearchSuggestion): Suggestion | undef
                     type: NonFilterSuggestionType.dir,
                     value: '^' + escapeRegExp(item.path),
                     description: descriptionParts.join(' — '),
-                    url: `${item.url}?suggestion`,
+                    url: `${item.url}?subtree`,
```

Comment:
I think it would be good to have this be `subtree=true` now too for consistency
```suggestion
                    url: `${item.url}?subtree=true`,
```